File: JavaProgLang/ArrayJava/InsertionInArray.java

import java.util.*;
public class InsertionInArray {
    public static void main(String args[]){
        int newArr[]=new int[5];
        Scanner s=new Scanner(System.in);
        System.out.println("Enter the elements into the array: ");
        for(int i=0;i<=4;i++){
            newArr[i]=s.nextInt();
        }
        System.out.println("The array you created is: ");
        for(int i=0;i<=4;i++){
            System.out.print(newArr[i]+" ");
        }

    }
}

File: C/Array/InsertionInArray.c
#include<stdio.h>
int main(){
    printf("Enter the data into the array: ");
    int newArr[5];
    for(int i=0;i<=4;i++){
        scanf("%d",&newArr[i]);
    }
    printf("\nThe array you created : ");
    for(int i=0;i<=4;i++){
        printf("%d ",newArr[i]);
    }
    return 0;
}